types/sinon/index.d.ts

```diff
@@ -1661,7 +1657,7 @@ declare namespace Sinon {
         expectation: SinonExpectationStatic;
 
         clock: {
-            create(now: number | Date): SinonFakeTimers;
+            create(now: number | Date): FakeTimers.Clock;
```

Comment:
this correctly will NOT have a `restore()` method - clocks made via `clock.create()` are "raw" fake-timers clocks which have no `restore` method (but do have `uninstall`).

clocks made via `useFakeTimers` in sinon have `restore` set to `clock.uninstall` (as an alias).